app/views/qsets/classpage.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="container">
+  <br>
+  <h3 class="center">
+      <%= render "qset_context" %>
+  </h3>
+
+  <%= render :file => 'qsets/index.html.erb' %>
```

Comment:
let's change `index.html.erb` into a partial template, since we're effectively using it as a partial, and `render` it the same way we `render` the question_list, all_mine_other_filter, etc.